### DISCUSSION TITLE
Improve tab's icon and label alignment

### DISCUSSION THREAD

**Discussion by Angelk90** [zed-industries/zed#22066] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

| Zed | Vscode | Webstorm |
|--------|--------|--------|
| <img width="174" alt="Screenshot 2024-12-07 alle 11 25 39" src="https://github.com/user-attachments/assets/b963a90b-6c1c-42b2-82e8-42bd993ed4b9"> |<img width="180" alt="Screenshot 2024-12-07 alle 11 26 06" src="https://github.com/user-attachments/assets/127a4ab9-72ba-4812-a565-c3a4430e570a"> | <img width="144" alt="Screenshot 2024-12-07 alle 11 26 23" src="https://github.com/user-attachments/assets/94a29a41-8aba-4b9c-ba96-1d3b79e6b2b9"> | 

As you can see from the images in this case the icon is aligned to the letter `a`, but as you can see in webstorm maybe it would be more correct to align it to the `p` and `g`, that is characters that have a lower base.

Is the zed icon aligned to the top of the text?

@danilo-leal : What do you think?

### Environment

```
Zed: v0.165.3 (Zed Preview)
OS: macOS 14.1.1
Memory: 16 GiB
Architecture: x86_64
```

**Comment by danilo-leal** [comment]
Hey! Thanks for flagging this; there are a lot of improvements we want to make to the tab component. My intention is to soon tackle them all at once. But technically no, the item isn't aligned at baseline now, it's just that the icon container is glued to the top of the container that holds the icon and label.
